settings/security/third-party/saml.rst

```diff
@@ -107,27 +111,34 @@ Keycloak
 Step 2: Configure Zammad
 ------------------------
 
-Enable SAML and enter your IdP’s details in the Admin Panel under
+Enable SAML and enter your IdP's details in the Admin Panel under
 **Settings > Security > Third Party Applications > Authentication via SAML**:
 
 .. image:: /images/settings/security/third-party/saml/zammad_connect_saml_thirdparty.png
    :alt: Example configuration of SAML
+   :scale: 60%
+   :align: center
 
 Display name
    Allows you to define a custom button name for SAML. This helps your users
    to understand better what the button on the login page does.
 
    Defaults to ``SAML``.
 
-IDP SSO Target URL
+IDP SSO target URL
    This is the target URL Zammad shall redirect to when the user presses
    the SAML button. For Keycloak, this needs to look like https://your.domain/realms/your-realm/protocol/saml
 
-IDP Certificate
+IDP single logout target URL
+   This is the target URL Zammad connects to when the user logs out. This is
+   required because otherwise just the session would be killed and no real
+   logout would be performed.
```

Comment:
I suggest something else: `This is the URL to which the single logout request and response should be sent.` All the other information is not needed IMO.